simulator/src/main/java/com/github/benmanes/caffeine/cache/simulator/policy/irr/ClockProPolicy.java

```diff
@@ -45,22 +44,20 @@
  * Effective Improvement of the CLOCK Replacement</a> and
  * <a href="http://www.slideshare.net/huliang64/clockpro">Clock-Pro: An Effective Replacement in OS
  * Kernel</a>.
- * <p>
- * This implementation is based on
- * <a href="https://bitbucket.org/SamiLehtinen/pyclockpro">PyClockPro</a> by Sami Lehtinen,
- * available under the MIT license.
  *
  * @author ben.manes@gmail.com (Ben Manes)
```

Comment:
Please update yourself to be the author 😃

Reply:
Done. 😃